trunk/src/modules/par_mesh.F90

```diff
@@ -129,7 +141,7 @@ subroutine distr_mesh()
                write(6,130) '[', RANK, ']: ',   &
                   'Receiving data from [',subd,'], nod = ',nod
             endif
-            count = nrdof_nod; src = subd; tag = nod
+            count = nrdof_nod; src = subd; tag = mod(nod,200000)
```

Comment:
This is probably appearing in some other parts of the code. How large are the tags allowed to be? Integer limit?

Reply:
It was in `update_gdof` and `update_Ddof` as well; I have it updated throughout my code but I'm not sure they've all made it in. It's only guaranteed to be a bit of over 200k (2^18) if I remember correctly. The limit used to be 2 million for Stampede2 but on Frontera the limit appears to be the minimum 2^18.

Reply:
Intel states `If a code exceeds the tag value of 32767 or 15 bits, the actual upper limit must be queried in the application. Otherwise, it creates a portability risk.`
Should we introduce a max tag parameter in the mpi_wrapper module, set it to a currently safe number, and use it throughout?

Reply:
I'll do this in the wrapper changes I have planned.

Reply:
Sounds good!